public_website/src/ui/components/footer/FooterList.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export type FooterListProps = {
+  title: string
+  listItems: { label: string; href: string }[]
+}
+
+export function FooterList({ title, listItems }: FooterListProps) {
+  return (
+    <StyledFooterList>
+      <h3>{title}</h3>
+      <ul>
+        {listItems.map((el, i) => {
+          return (
+            <li key={i}>
+              <a href={el.href}>{el.label}</a>
+            </li>
+          )
+        })}
```

Comment:
`el` et `i` ne sont pas très explicite. 
Et je crois que l'utilisation de l'index n'est pas recommandé pour le key (ça va surement remonter dans Sonar).
On peut utiliser anchor.label si ils sont tous différent

```suggestion
        {listItems.map((anchor) => {
          return (
            <li key={anchor.label}>
              <a href={anchor.href}>{anchor.label}</a>
            </li>
          )
        })}
```